landing/tests/tests.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from django.test import Client
```

Comment:
It is not the correct way to use client fixtures in pytest
Please take a look here
https://pytest-django.readthedocs.io/en/latest/helpers.html#id2

Do not confuse with this 
https://docs.djangoproject.com/en/4.2/topics/testing/tools/

Read the pytest-django docs, it is wrapping the django-test client